packages/wallet-standard/src/initialize.ts

```diff
@@ -0,0 +1,7 @@
+import type { GlowAdapter } from "@glow-xyz/glow-client";
+import { DEPRECATED_registerWallet } from "./register.js";
+import { GlowWallet } from "./wallet.js";
+
+export function initialize(glow: GlowAdapter): void {
+  DEPRECATED_registerWallet(new GlowWallet(glow));
```

Comment:
Here we call the deprecated version of this function internally. Will explain below.

Reply:
Could we instead just not support the deprecated stuff? This standard was just released so I can't imagine there are many apps on the old versions?